starwars/src/components/StarWarsCharacterList.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import StarWarsCharacter from './StarWarsCharacter';
+
+
+const StarWarsCharacterList = ({starwarsChars}) => (
+    <div className="characterList">
+      {starwarsChars.map(character =>
+            <StarWarsCharacter name={character.name} key={character.created} birthYear={character.birth_year} height={character.height} mass={character.mass} gender={character.gender} />
```

Comment:
If you are passing every single character in as props you can use the spread operator ```{...character}```